### ISSUE TITLE
Warning messages when no external JVM is installed

### ISSUE THREAD

**Issue by jamiesnape** [bazelbuild/homebrew-tap#2]
For example,
```
$ bazel version
Unable to find any JVMs matching version "(null)".
No Java runtime present, try --request to install.
Unable to find any JVMs matching version "1.7+".
No Java runtime present, try --request to install.
Build label: 0.16.1
Build target: bazel-out/darwin-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Mon Aug 13 13:42:50 2018 (1534167770)
Build timestamp: 1534167770
Build timestamp as int: 1534167770
```
Warning is coming from the file generated here: https://github.com/bazelbuild/homebrew-tap/blob/f74c06d80a84770c7e42767384ac35a0b49fa99d/bazel.rb#L29

Within that file, if `JAVA_HOME` is not set, `/usr/libexec/java_home` is run, which needs a JVM.

Otherwise Bazel runs all commands successful and compiles Java targets successful, so clearly no external JVM is really needed.

**Comment by jamiesnape** [comment]
Incidentally, most likely the reason for this issue: https://github.com/bazelbuild/bazel/issues/668.

**Comment by agikoo** [comment]
bazelbuild/bazel#668 has been closed, with cannot repro.

I install by Bazel Documentation(Installing Bazel on macOS - use Homebrew), and it doesn't say need install JVM first. So want to use bazel, JVM is required or not?

**Comment by buchgr** [comment]
Thanks for reporting this issue! Bazel does not require a system JDK installed. The warning is benign. Here's a fix https://github.com/bazelbuild/bazel/pull/5992

**Comment by buchgr** [comment]
Fixed upstream as f2049548b07
